frontend/src/metabase/timelines/collections/components/TimelineCard/TimelineCard.styled.tsx

```diff
@@ -30,33 +30,39 @@ export const CardDescription = styled.span`
   word-wrap: break-word;
 `;
 
-export interface CardAsideProps {
+export interface CardCountProps {
   isTopAligned?: boolean;
 }
 
-export const CardAside = styled.span<CardAsideProps>`
+export const CardCount = styled.span<CardCountProps>`
   display: block;
   flex: 0 0 auto;
   color: ${color("text-dark")};
   align-self: ${props => (props.isTopAligned ? "flex-start" : "")};
 `;
 
+export const CardMenu = styled.span`
+  display: block;
+  flex: 0 0 auto;
+`;
+
 export const CardRoot = styled(Link)`
   display: flex;
   padding: 1.75rem;
   align-items: center;
   border: 1px solid ${color("border")};
   border-radius: 6px;
+  cursor: ${props => (props.to ? "pointer" : "default")};
 
   &:hover {
-    border-color: ${color("brand")};
+    border-color: ${props => (props.to ? color("brand") : "")};
 
     ${CardIcon} {
-      color: ${color("brand")};
+      color: ${props => (props.to ? color("brand") : "")};
     }
 
     ${CardTitle} {
-      color: ${color("brand")};
+      color: ${props => (props.to ? color("brand") : "")};
```

Comment:
Looks like all the `hover` styles are needed only when `CardRoot` has a `to` prop. Should we maybe extract this block of CSS and add it conditionally? Meaning something like:

```jsx
import { css } from "@emotion/react"

const cardRootHoverStyle = css`/* all the hover styles here */`

export const CardRoot = styled(Link)`
   /* common styles */
   ${props => props.to && cardRootHoverStyle}
`
```

Reply:
Agree, fixed